stack.yaml

```diff
@@ -132,6 +132,12 @@ packages:
     git: https://github.com/input-output-hk/log-warper.git
     commit: b8c390357b27e7ae3b9b0c5309bf9d0bbbf09852
   extra-dep: true
+# patched version of servant-quickcheck that allows us to use a custom manager
+# and also handles mvar appropriately
+- location:
+    git: https://github.com/parsonsmatt/servant-quickcheck
+    commit: 38cd86bd2214dcf623b216f90d8fc9657549aed6
+  extra-dep: true
```

Comment:
ah! we can use hackage version instead :) `servant-quickcheck-0.0.4.1` has this exposed: http://hackage.haskell.org/package/servant-quickcheck-0.0.4.1/docs/Servant-QuickCheck-Internal-QuickCheck.html#v:serverSatisfiesMgr